client/app/screens/Loading.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { Button, Image, Linking } from 'react-native';
+import Login from 'react-native-login';
+
+import {
+  Body,
+  Card,
+  CardItem,
+  Col,
+  Container,
+  Content,
+  Grid,
+  H3,
+  Icon,
+  ListItem,
+  Row,
+  Text,
+  variables,
+  View,
+} from 'native-base';
+
+const config = {
+  url: 'https://chat.sagaoftherealms.net/auth',
+  realm: 'demo',
+  client_id: 'Android',
+  redirect_uri: 'keycloak-demo://app',
+};
+
+export default class LoadingScreen extends React.Component {
+  static navigationOptions = {
+    title: 'Loading',
+  };
+  constructor(props) {
+    super(props);
+    Login.conf = config;
+    this.state = {
+      isLoading: true,
+      hasInitialUrl: null,
+      hasToken: null,
+      token: '',
+    };
+  }
+
+  checkTokens() {
```

Comment:
I've been prefixing our private methods with `_` to differentiate them from React's API, do you mind doing the same here for consistency?